### PR TITLE
feat: notify claim event (CMC-1207)

### PR DESCRIPTION
**Pull request by swalker125**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CMC-1207

### Change description ###

Notify defendant of claim event
Awaiting case notification state

New state flow PENDING_CASE_ISSUED -> AWAITING_CASE_NOTIFICATION -> CREATED

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/unspec/handler/callback/user/NotifyClaimCallbackHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                .containsOnly(NOTIFY_DEFENDANT_OF_CLAIM.name(), "READY");
+        }
+    }
+
+    @Nested
+    class SubmittedCallback {
+
+        public static final String CONFIRMATION_BODY = "<br />What happens next\n\n"
+            + "The defendant legal representative's organisation has been notified and granted access to this "
+            + "claim.\n\n"
```
Comment (sabahirfan) [inline src/test/java/uk/gov/hmcts/reform/unspec/handler/callback/user/NotifyClaimCallbackHandlerTest.java:53]: Shouldn't we use `%n` instead of `\n` ?

Comment (swalker125) [reply src/test/java/uk/gov/hmcts/reform/unspec/handler/callback/user/NotifyClaimCallbackHandlerTest.java:53]: Hmm, seems we have places we use %n and places we use \n. I will leave as is, but maybe we should have tech debt to clean up?

**Review by sabahirfan — DISMISSED** [review]

File: src/test/java/uk/gov/hmcts/reform/unspec/handler/callback/user/NotifyClaimCallbackHandlerTest.java (hunk 1 above)
Comment (sabahirfan) [reply src/test/java/uk/gov/hmcts/reform/unspec/handler/callback/user/NotifyClaimCallbackHandlerTest.java:53]: We did clean that few months back. Looks like it's time again to clean this one. :)

**Review by ckalista — DISMISSED** [review]
👍 good stuff

**Review by douglasrice — APPROVED** [review]

**Review by sabahirfan — APPROVED** [review]

**Pull request merged**